konrad/core.py

```diff
@@ -293,11 +329,11 @@ def run(self):
                     n0 - self.radiation['toa'][0]) / (A * tau
                                                       ) * self.atmosphere['CO2']
 
-            # Save the old temperature profile. They are compared with
-            # adjusted values to check if the model has converged.
-            T = self.atmosphere['T'].copy()
+            # Save the old temperature profile. It is compared with adjusted
+            # values to check if the model has converged.
+            self.oldT = self.atmosphere['T'][0].copy()
```

Comment:
Did you remove the check for the temperature convergence completely?

Reply:
The temperature difference is still used by the timestep adjuster. I do not want to touch the timestep adjuster for now.